core/broadleaf-framework-web/src/main/java/org/broadleafcommerce/core/web/checkout/model/BillingInfoForm.java

```diff
@@ -104,12 +103,4 @@ public String getPaymentName() {
     public void setPaymentName(String paymentName) {
         this.paymentName = paymentName;
     }
-
-    public String getEmailAddress() {
-        return emailAddress;
-    }
-
-    public void setEmailAddress(String emailAddress) {
-        this.emailAddress = emailAddress;
-    }
```

Comment:
Note: I had added this property in a previously merged PR. This change is simply reverting back to the original state of the BillingInfoForm.